Packer/ArkCase-CE/startup.service

```diff
@@ -7,7 +7,7 @@ Requires=samba.service
 Type=simple
 User=root
 Group=root
-ExecStart=/usr/local/bin/startup.sh
+ExecStart=/bin/bash /usr/local/bin/startup.sh
```

Comment:
This is not necessary, since /usr/local/bin/startup.sh has mode 755 and starts with the line `#!/bin/bash`